js/script.js

```diff
@@ -5,7 +5,145 @@ const barBottom = document.querySelector('.bar-bottom');
 const barMiddle = document.querySelector('.bar-middle');
 const navList = document.querySelectorAll('.nav-link');
 const logo = document.querySelector('#logo');
+const cardContainer = document.querySelector('.card-container');
+const portfolio = document.querySelector('#portfolio');
 
+// Card objects for array
+
+const card1 = {
+  extraClasses: ['card1'], title: null, description: null, featuredIMGURL: null, technologies: null, demoURL: '#', linkToSource: '#', button: true,
+};
+
+const card2 = {
+  extraClasses: ['card2'], title: 'Professional Art Printing Data', description: 'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry\'s standard', featuredIMGURL: null, technologies: ['CSS', 'Bootsrap', 'Ruby'], demoURL: '#', linkToSource: '#', button: null,
+};
+
+const card3 = {
+  extraClasses: ['card3'], title: 'Professional Art Printing Data', description: 'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry\'s standard', featuredIMGURL: null, technologies: ['CSS', 'Bootsrap', 'Ruby'], demoURL: '#', linkToSource: '#', button: null,
+};
+
+const card4 = {
+  extraClasses: null, title: 'Professional Art Printing Data', description: 'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry\'s standard', featuredIMGURL: null, technologies: ['CSS', 'Bootsrap', 'Ruby'], demoURL: '#', linkToSource: '#', button: null,
+};
+
+const card5 = {
+  extraClasses: ['card2'],
+  title: 'Professional Art Printing Data',
+  description: 'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry\'s standard',
+  featuredIMGURL: null,
+  technologies: ['CSS', 'Bootsrap', 'Ruby'],
+  demoURL: '#',
+  linkToSource: '#',
+  button: null,
+};
+
+const card6 = {
+  extraClasses: ['card3'],
+  title: 'Professional Art Printing Data',
+  description: 'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry\'s standard',
+  featuredIMGURL: null,
+  technologies: ['CSS', 'Bootsrap', 'Ruby'],
+  demoURL: '#',
+  linkToSource: '#',
+  button: null,
+};
+
+const cardArray = [card1, card2, card3, card4, card5, card6];
+
```

Comment:
[OPTIONAL]: Great implementation here. I think your code would have been more readable if you had implemented you `array of object` like this:

```javascript
const projects = [{
    name: "Submarine game",
    description: "A Phaser3 platform game built as part of the Microverse JavaScript program.",
    image: "link_to_image.jpg",
    technologies: ["Phaser3", "JavaScript", "Jest"],
    link: "link_to_live_version.com",
}]
```

Reply:
@micheaol 

Thank you this is useful and I will consider adding it when I have the time!